01week/bio/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Samantha's Bio Page</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+<div id="wrapper">
+
+	<div id="header">
+      <img src="https://unsplash.com/?photo=21ae7tGpJtE"
+		<div id="nav"><a href="index.html">Home</a><br></br><a href="#">About Me</a><br></br>
+      <a href="#">Contact Me</a<br></br>
+      </div>
+		<div id="bg"></div>
+<h2>
+	<div id="main-content">
+		<div id="left-column">
+			<div id="logo">
+			<h1>Samantha Leal</h1>
+			</div>
+			<div class="box">
+      </h2>
+        		<h1>Who am I?</h1>
+        		<p>This is just a fraction of me!</p>
+            <h3>
+				<ul style="margin:10px 20px 10px 20px;">
```

Comment:
If you link this html file to your bio css file you can do away with the margin property and value in your html.  Just make a selector in your css with those properties and values for your html to refer to.